JitsiConferenceEvents.ts

```diff
@@ -153,6 +153,16 @@ export enum JitsiConferenceEvents {
      */
     LAST_N_ENDPOINTS_CHANGED = 'conference.lastNEndpointsChanged',
 
+    /**
+     * The forwarded sources set is changed.
+     *
+     * @param {Array<string>|null} leavingForwardedSources the sourceNames of all the tracks which are leaving forwarded
```

Comment:
For both params, the type is always an array (not null)

Reply:
👍  updated